frontend/src/pages/ErrorDetails/index.tsx

```diff
@@ -81,7 +84,18 @@ function ErrorDetails(): JSX.Element {
 	}
 
 	if (status === 'error' || ErrorIdStatus === 'error') {
-		return <Typography>{data?.error || errorIdPayload?.error}</Typography>;
+		return (
+			<Typography>
+				{data?.error || errorIdPayload?.error || defaultError}
+			</Typography>
+		);
+	}
+
+	if (
+		(status === 'success' && data?.statusCode !== 200) ||
```

Comment:
Same here status code >= 400

Reply:
not sure why we are going with >=400 ... if data is successfully fetched that’s only be applicable for 200 if not that's an error

Reply:
ok

Reply:
is there any other use case ... that i am not not able to think here ?

Reply:
Or any other redirect in 300 series _(with Axolo)_

Reply:
cool will add _(with Axolo)_